Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -1211,14 +1211,14 @@ GDCMImageIO::Write(const void * buffer)
       outpixeltype.SetPixelRepresentation(static_cast<unsigned short>(std::stoi(pixelRep.c_str())));
       if (this->GetNumberOfComponents() != 1)
       {
-        itkExceptionMacro(<< "Sorry Dave I can't do that");
+        itkExceptionMacro("Sorry Dave I can't do that");
```

Comment:
“I'm sorry, Dave. I'm afraid I can't do that,” said HAL 9000
(c) 2001: A Space Odyssey

Reply:
Thanks for the reference, Mihail! Honestly I doubt if such an error message is very helpful to the user. In general, I'm not a fan of "funny" error messages. Unless they really help to communicate to the user what's going wrong. But anyway, that's beyond the scope of this pull request!

Reply:
Maybe it is an error message which is not supposed to be reachable by the GDCM's users?

Reply:
Maybe... but then I guess the message only for someone named Dave 😸  Thanks for merging @dzenanz !